api/src/pcapi/routes/public/documentation_constants/descriptions.py

```diff
@@ -9,6 +9,11 @@
 - Dates of event offers are stored in the **UTC format** to be able to format them correctly, according to the user timezone, in our application interface.
 - Any non-blank field sent using a `PATCH` method will be considered as changed, even if the new value is equal to old value.
   _For example, if you update the stock of an event, you **should not resend the `beginningDate`** if it has not changed because, otherwise it is going to trigger the reschedule process on our side._
+
+# Authentication
+The authentication system on Pass Culture API is using an API Key. For each of your requests, you should put your API key in the authorization header of the request following the `Bearer`pattern.
+That is to say that in your request headers you should have a line that look like this : `Authorization: Bearer {you_api_key}` (**⚠️ Make sure to use the right API key for the right environment**)
+An API key is linked to a provider and will give you access to the venues linked to this provider by the offerers.
```

Comment:
(je suis un peu perdu dans cette phrase)
This API key gives you access to the venues that have allowed you to use our API.

Reply:
en fait j'essayais d'expliquer que la clé API authentifie un provider et ensuite que le provider est lié à des venues. 
Je voulais expliciter le fait que la clé API n'était pas liée directement aux lieux.